ee/maintained-apps/outputs/teamviewer/windows.json

```diff
@@ -0,0 +1,19 @@
+{
+  "versions": [
+    {
+      "version": "15.64.3",
+      "queries": {
+        "exists": "SELECT 1 FROM programs WHERE name = 'TeamViewer' AND publisher = 'TeamViewer';"
+      },
+      "installer_url": "https://download.teamviewer.com/download/version_15x/TeamViewer_Setup_x64_15.64.3.exe",
+      "unique_identifier": "TeamViewer",
+      "install_script_ref": "45f422bf",
+      "uninstall_script_ref": "a13b74d9",
+      "sha256": "2a8aefd4f2e4a44c3f7c55caa35d61827e37392fe72726b9a980299f9199a230"
+    }
+  ],
+  "refs": {
+    "45f422bf": "# Learn more about .exe install scripts:\n# http://fleetdm.com/learn-more-about/exe-install-scripts\n\n$exeFilePath = \"${env:INSTALLER_PATH}\"\n\ntry {\n\n# Add argument to install silently\n# Argument to make install silent depends on installer,\n# each installer might use different argument (usually it's \"/S\" or \"/s\")\n$processOptions = @{\n  FilePath = \"$exeFilePath\"\n  ArgumentList = \"/S\"\n  PassThru = $true\n  Wait = $true\n}\n    \n# Start process and track exit code\n$process = Start-Process @processOptions\n$exitCode = $process.ExitCode\n\n# Prints the exit code\nWrite-Host \"Install exit code: $exitCode\"\nExit $exitCode\n\n} catch {\n  Write-Host \"Error: $_\"\n  Exit 1\n}\n",
+    "a13b74d9": "$product_code = \n\n# Fleet uninstalls app using product code that's extracted on upload\nmsiexec /quiet /x $product_code\nExit $LASTEXITCODE\n"
```

Comment:
@mostlikelee nuts looks like this is missing a product code

Reply:
argh yeah looks like no product code: https://github.com/microsoft/winget-pkgs/blob/master/manifests/t/TeamViewer/TeamViewer/15.64.3/TeamViewer.TeamViewer.installer.yaml